config.template

```diff
@@ -105,6 +127,7 @@ regionIndicesToPlot = [6]
 N_movavg = 12
 
 [nino34_timeseries]
+
 ## compare to output from another model run?
```

Comment:
Why the extra spaces?  I think it is probably clearer to have sections as a continuous, non-spaced block.

```python
[section]
# discussion of what this section is
option1 = 1
optionA = 'A'
```